relays/ethereum/src/ethereum_client.rs

```diff
@@ -32,12 +32,12 @@ use jsonrpsee::transport::http::HttpTransportClient;
 use jsonrpsee::Client;
 use parity_crypto::publickey::KeyPair;
 
-use std::collections::{HashSet, VecDeque};
+use std::collections::HashSet;
 
 // to encode/decode contract calls
 ethabi_contract::use_contract!(bridge_contract, "res/substrate-bridge-abi.json");
 
-type Result<T> = std::result::Result<T, RpcError>;
+type RpcResult<T> = std::result::Result<T, RpcError>;
```

Comment:
If we're gonna change this in the Ethereum client we should make the same change in the Substrate client for consistency